### DISCUSSION TITLE
2023 Ann Arbor data

### DISCUSSION THREAD

**Discussion by vielmetti** [fenggroup/bike-traffic-plotly-dash#3] [Announcements]
What are the current plans for Ann Arbor data for 2023 (additional locations, starting to publish data etc?) I saw the box and the rumble strips on Division.

**Comment by frfeng** [comment]
We just put up the counter on Division on May 1, and the first week's data is already up [on the dashboard](https://bikecounter.org/annarbor-1). We hope to update the data weekly or as often as we have time to :) 

We have one additional counter that I have yet decided where to put it given that the DDA plans to [install permanent counters for each downtown bikeway this year](https://www.mlive.com/news/ann-arbor/2023/03/ann-arbors-new-bikeways-used-by-many-help-grow-bike-ridership.html). I'm currently leaning towards putting ours somewhere on Packard between State and E Stadium.
